### PR TITLE
Question page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

**Comment by shruthipatil11** [comment]
looks great to merge, 
Meta tag having author ,page description could be added in head section

**Review by SinchanaMS — APPROVED** [review]
The code looks good.
In the UI, when you hover on "Next", the content of the quespage-container is slightly shifting, please check that.

File: common.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{
+    background-color: var(--main-background-color);
+    box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
+    font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
+}
+*
+{
+    padding: 0;
+    margin: 0;
+}
```
Comment (ashishkatiyar2404) [inline common.css]: root variable is always on top of all the CSS.

File: question.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    padding: 2rem;
+}
+.quiz-timer{
+    display: flex;
+    width: 7rem;
+    height: 3rem;
+    background-color: var(--tertiary-card-color);
+    align-items: center;
+    justify-content: space-between;
+    padding: 2px 3px;
```
Comment (ashishkatiyar2404) [inline question.css]: you can increase left and right padding here
  #padding: 2px 6px

File: question.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    display: block;
+    border: 1px solid var(--divider-bg-color);
+    border-radius: 10px;
+    padding: 1rem;
+    margin-bottom: .5rem;
+}
+.quiz-question input{
+    display: none;
+}
+.quiz-question label:hover, .quiz input:checked + label {
```
Comment (ashishkatiyar2404) [inline question.css]: good to see you are using adjacent sibling selector nice.💯

File: question.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+           <label for="option-a">Gucci</label>
+           <input type="radio" name="answer" id="option-b">
+           <label for="option-b">Christian Louboutin</label>
+           <input type="radio" name="answer" id="option-c">
+           <label for="option-c">Jimmy Choo</label>
+           <input type="radio" name="answer" id="option-d">
+           <label for="option-d">Louis Vitton</label>
+       </main>
+       <div class="ques-btn-container">
+        <a class="primary-button-solid" href="#">Next</a>
```
Comment (ashishkatiyar2404) [inline question.html]: instead of using an anchor tag, you can use a button tag for your Next button(for semantic)

**Review by ashishkatiyar2404 — APPROVED** [review]
Suggested some changes please do that.
Other its looks good to merge your UI is also good. The  BG gradient is looking good💯
One more thing you can change is that on hovering the Next button your .ques-page div container size increases a little bit. Please fix that too.

**Comment by ShrutiLakhani** [comment]
> looks great to merge, Meta tag having author ,page description could be added in head section

I am not aware of this meta tag. I will check later and will implement. Thanks for the suggestion

File: common.css (hunk 1 above)
Comment (ShrutiLakhani) [reply common.css]: Thanks. Will make that change

**Comment by ShrutiLakhani** [comment]
> The code looks good. In the UI, when you hover on "Next", the content of the quespage-container is slightly shifting, please check that.

Sure. I will check that. Thanks

**Comment by ShrutiLakhani** [comment]
> 
Made changes

**Pull request merged**